storage/innobase/xtrabackup/test/inc/keyring_common.sh

```diff
@@ -8,7 +8,11 @@ function test_do()
 {
 	table_options=$1
 	transition_key=$2
-
+  keyring_type=$3
+  vlog "-- running test with: -- "
+  vlog "-- table_options: ${table_options} --"
+  vlog "-- transition_key: ${transition_key} --"
+  vlog "-- keyring_type: ${keyring_type} --"
```

Comment:
alignment looks out of place, should it be tabs?

Reply:
@satya-bodapati file itself is wrong, we should be using two spaces instead of tabs at 8.0 . Shall we fix the whole file ?

Reply:
well better to adjust the current file. Aslo the rule for 8.0 is mostly for C++ sources (clang-format). sh files we are free to use any style.